crutest/src/cli.rs

```diff
@@ -742,10 +742,20 @@ async fn process_cli_command(
     verify_output: Option<PathBuf>,
 ) -> Result<()> {
     match cmd {
-        CliMessage::Activate(gen) => match guest.activate_with_gen(gen).await {
-            Ok(_) => fw.send(CliMessage::DoneOk).await,
-            Err(e) => fw.send(CliMessage::Error(e)).await,
-        },
+        CliMessage::Activate(gen) => {
+            let gc = guest.clone();
+            let _handle = tokio::spawn(async move {
+                match gc.activate_with_gen(gen).await {
+                    Ok(_) => {
+                        println!("Activate Successful");
+                    }
+                    Err(e) => {
+                        println!("Activate failed: {:?}", e);
+                    }
+                }
+            });
+            fw.send(CliMessage::DoneOk).await
```

Comment:
Can you add a comment explaining why it's fine to return `DoneOk` even though the activation may not be complete?  Or should we leave `CliMessage::Activate` unchanged and add `CliMessage::StartActivate` to start activation but not wait for it?

Reply:
I'm also for having a separate command to spawn an activation task

Reply:
I'll both comment and make a new command :)

Reply:
Done in 4f9da70

Here is how the new CLI looks:
```
>> help
Usage: <COMMAND>

Commands:
  activate          Send an activation message and wait for an answer
  activate-request  Send an activation message and don't wait for an answer
  commit            Commit the current write_log data to the minimum
                        expected counts
  deactivate        Deactivate the upstairs
  dsc               DSC
  expected          Report the expected read count for an offset
  export            Export the current write count to the verify out file
  fill              Run the fill then verify test
  flush             Flush
  generic           Run Generic workload
  info              Request region information
  is-active         Report if the Upstairs is ready for guest IO
  perf              Run the client perf test
  quit              Quit the CLI
  read              Read from a given block offset
  replace
  rr                Issue a random read
  rw                Issue a random write
  show              Show the work queues
  wait              Change the wait state between true and false
  verify            Read and verify the whole volume
  write             Write to a given block offset
  write-unwritten   WriteUnwritten to a given block offset. Note that the
                        cli will decide if a block is written to based on the
                        cli's internal write log.  This log may not match if the
                        block actually has or has not been written to
  uuid              Get the upstairs UUID
  help              Print this message or the help of the given
                        subcommand(s)

Options:
  -h, --help  Print help

>> activate -g 2
Ok
>> info
Got info: 4096 100 122880000
>> is-active
Active is: true
>> deactivate
Ok
>> dsc stop -c 0
Got res: Ok(())
>> activate-request -g 3
Ok
>> is-active
Active is: false
>> dsc start -c 0
Got res: Ok(())
>> is-active
Active is: true
>> 
```